desc/geometry/curve.py

```diff
@@ -671,6 +671,7 @@ def __init__(
         knots=None,
         method="cubic",
         name="",
+        discontinuous_knots=None,
```

Comment:
instead of knot values probably easier to have the user give the indices where its allowed to break, then you can avoid the `find_nearest` thing below